trieste/models/gpflow/models.py

```diff
@@ -52,6 +59,7 @@ def __init__(
         """
         super().__init__(optimizer)
         self._model = model
+        self._posterior = self._model.posterior()  # cache posterior for fast sequential predictions
```

Comment:
Note that there is a issue within GPflow that says that the posterior is broken for GPR (I don't know its merit but there is a PR to fix it so I guess it is indeed broken!)

https://github.com/GPflow/GPflow/issues/1733
https://github.com/GPflow/GPflow/pull/1734

I thought alpha was *supposed to be* equal to y, but I'm not an expert.

Reply:
that PR was merged into develop branch today :)